pre_commit/main.py

```diff
@@ -94,6 +94,11 @@ def main(argv=None):
         '--source', '-s',
         help='The remote branch"s commit_id when using `git push`',
     )
+    run_parser.add_argument(
+        '--allow-unstaged-config', default=False, action='store_true',
+        help='Allow an unstaged config to be present.  Note that this will \
```

Comment:
nit: prefer paren'd continuation to `\`s

I'll fix it when I merge if you don't want to bother :)